frontend/README.md

```diff
@@ -0,0 +1,12 @@
+# How to setup frontend
```

Comment:
Please, move this into review which is in repo root folder, so we will keep all installation instructions for both frontend and backend